src/request_test.go

```diff
@@ -289,7 +289,7 @@ func TestHandleResponse(t *testing.T) {
 
 func TestDefaultMetricType(t *testing.T) {
 	eventType := "TestSample"
-	defs, err := parseYaml("../test/activemq.yml")
+	defs, err := parseYaml("../tests/data/activemq.yml")
```

Comment:
extracting the assets to `data/` is cool, but any reason to rename test to tests?
`test/` is a de-facto standard https://github.com/golang-standards/project-layout/tree/master/test